packages/insured-bridge-relayer/src/CrossDomainFinalizer.ts

```diff
@@ -52,7 +52,9 @@ export class CrossDomainFinalizer {
     // we don't want to finalize DAI actions due to this not working over the canonical Optimism bridge.
     const whitelistedL2Tokens = this.l1Client
       .getWhitelistedL2TokensForChainId(this.l2Client.chainId.toString())
-      .filter((address) => address !== "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1"); // Remove DAI on Optimism address.
+      .filter((address) =>
+        this.l2Client.chainId != 10 ? true : address !== "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1"
```

Comment:
nit use !== to be consistent instead of !=

Reply:
In fact, maybe just put the whole chunk of code into a helper function since its copied exactly twice

Reply:
sure

Reply:
done